billboard.py

```diff
@@ -314,7 +324,43 @@ def getMinistatsCellValue(fieldName, ifNoValue=None):
             self.entries.append(entry)
 
     def _parseNewStylePage(self, soup):
-        dateElement = soup.select_one("button.date-selector__button.button--link")
+        artists = []
+        # extract artists info from page
+        pageContent = json.loads(soup.select_one("div#charts")['data-charts'])
+        for artist in pageContent:
+            artistName = artist['artist_name']
+            artistImages = artist['artist_images']
+            titleImages = artist['title_images']
+            artistImage = None
+            titleImage = None
+            # searching for image in 3 different sizes
+            if artistImages:
+                _artistImage = artistImages['sizes'].get('original')
+                if not _artistImage:
+                    _artistImage = artistImages['sizes'].get(
+                        'ye-landing-lg-2x')
+                if not _artistImage:
+                    _artistImage = artistImages['sizes'].get(
+                        'ye-landing-med-2x')
+                artistImage = _artistImage['Name'] if _artistImage else None
+            if titleImages:
+                _titleImage = titleImages['sizes'].get('ye-landing-lg-2x')
```

Comment:
Why do you prefer `ye-landing-lg-2x` here but `original` for the artist image?

Reply:
for some cases, `ye-landing-lg-2x` was available when `original` wasn't, but the approach you suggested above will fix that.